pkg/didcomm/crypto/jwe/authcrypt/encrypt_test.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
We need test cases.

Reply:
I do have a test case that generates a JWE using the PHP example keys, but we can't compare the cipher output as it's based on cryptographically random keys/nonce generation

Reply:
pushed a patch with a comment... will definitely need to test interoperability with other libraries (like the reference php one: https://github.com/gamringer/php-authcrypt)